azure.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+name: chat-with-your-data-solution-accelerator
+metadata:
+  template: chat-with-your-data-solution-accelerator@0.0.1-beta
+
+services:
+  web:
+    project: ./code
+    language: py
+    host: appservice
+    hooks:
+      prepackage:
+        windows:
```

Comment:
I believe you need to add something like: 
```yaml
        posix:
          shell: sh
          run:  cd ./frontend;npm install;npm run build;
          interactive: true
          continueOnError: false
```

Reply:
thanks, are you running in devcontainer?

Reply:
yes